core-bundle/src/Command/UserListCommand.php

```diff
@@ -61,10 +61,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $columns = $input->getOption('column');
 
         switch ($input->getOption('format')) {
-            case 'text':
-                trigger_deprecation('contao/core-bundle', '4.13', 'Using --format=text is deprecated and will be removed in Contao 5. Use --format=txt instead.');
```

Comment:
The deprecation message is probably wrong, is it? You should use `--format=text` instead of `--format=text`? :upside_down_face:

Reply:
`--format=txt`

Reply:
seems correct to me (`format=txt`)